auction9-frontend/src/app/sidenav/sidenav.component.ts

```diff
@@ -15,20 +16,33 @@ export class SidenavComponent implements OnInit, OnDestroy {
 
   panelOpenState = false;
   mobileQuery: MediaQueryList;
+  isLoggedIn: boolean;
 
   private _mobileQueryListener: () => void;
 
   constructor(
     changeDetectorRef: ChangeDetectorRef,
     media: MediaMatcher,
-    private dialog: MatDialog
-  ) {
+    private dialog: MatDialog) {
     this.mobileQuery = media.matchMedia('(max-width: 600px)');
     this._mobileQueryListener = () => changeDetectorRef.detectChanges();
     this.mobileQuery.addEventListener('change', this._mobileQueryListener);
   }
 
-  ngOnInit() { }
+  ngOnInit() {
+    Auth.currentAuthenticatedUser()
```

Comment:
Probably this call with setting `isLoggedIn` variable should be in used for more components. Consider move it on the root app level, it'll we easier to have one central place where we check the user, instead of putting the same code into all component where we need it.

Reply:
Maybe for now it's not needed, but it will be when you start implementing visibility on other pages.